config/sync/block.block.fieldblock_page_section.yml

```diff
@@ -5,9 +5,9 @@ dependencies:
   config:
     - field.storage.node.field_page_section
   module:
-    - ctools
```

Comment:
Any thoughts on why this happened?

Reply:
I don't know exactly why, for whatever reason it seems that it's a causation of adjusting the block layout for the Lexington theme.  There are multiple places here where ctools is being interchanged with node. After testing the site is working as expected. So I presume ctools isn't a needed dependency for these blocks.